src/Data/Cursor/AbstractCursor.php

```diff
@@ -99,7 +108,7 @@ public function __construct( $query_vars, $cursor = 'after' ) {
 	 * @return mixed|null
 	 */
 	public function get_query_var( string $name ) {
-		return ! empty( $this->query_vars[ $name ] ) ? $this->query_vars[ $name ] : null;
+		return isset( $this->query_vars[ $name ] ) ? $this->query_vars[ $name ] : null;
```

Comment:
Confirming the intended change in behavior here, since this doesn't just return `false`, but empty strings and arrays. Alternatively we can use a string 'skip' value instead of an explicit 'false'.

Not sure what it is offhand, but we should follow whatever WP_Query does internally.

Reply:
This adds support for `[]` as a value

Reply:
@justlevine So `''`, `[]`, and `false` aren't valid values for SQL statements? Or just `''`

Reply:
@justlevine @jasonbahl Changes made 👍🏿

Reply:
My comment here is not about SQL statements, it's what WordPress/WPGraphQL considers as a set `query_var`, and what our code is expecting from `AbstractCursor::get_query_var()`.

- Before this change, WPGraphQL would treat _any_ falsey value as if it wasn't set.
- After this change, `''`, `[]` and `false`, are all returned instead of `null`. This is a change in behavior that _may_ be breaking (eg, `if ( isset( $cursor->get_query_var() )` now returns true).
I'm calling this out to ensure someone reviews all the uses of `AbstractCursor::get_query_var()` to confirm none of those possible return values cause the code to behave unexpectedly.
- WordPress core [treats an `''` value the same as if it's unset](https://github.com/WordPress/wordpress-develop/blob/6.3/src/wp-includes/class-wp-query.php#L1833-L1839), so at the bare minimum we should at least continue to return `null` in that case. I.e. `isset( $this->query_vars[ $name ] ) && $this->query_vars[ $name ] !== '' ) ? $this->query_vars[ $name ] : null`

Reply:
Understood. The proper changes have been made. 👍🏿